src/js/modules/rpc/service.ts

```diff
@@ -13,62 +13,80 @@ import { safeLoadAll } from "js-yaml";
 import { join, resolve } from "path";
 import * as fs from "fs";
 import { promisify } from "util";
-import { newLogger } from "../utilities/Logging";
+import { createLoggerFromLogFilePath } from "../utilities/Logging";
+import { Logger } from "winston";
 
-const logger = newLogger("service");
+class Service {
+  private logger: Logger;
 
-// read yaml config file
-const readConfigFile = (confPath: string): Promise<Record<string, any>> => {
-  try {
-    logger.info(`Reading from config file: ${confPath}`);
-    return fs.promises.readFile(confPath).then((file) => safeLoadAll(file)[0]);
-  } catch (e) {
-    return Promise.reject(new Error(`Error reading config file: ${e.message}`));
-  }
-};
+  constructor() {
+    const configPathArg = process.argv.splice(2)[0];
+    const defaultConfigPath = "/var/lib/redpanda/conf/redpanda.yaml";
+    const defaultCoprocessorPath = "/var/lib/redpanda/coprocessor";
```

Comment:
This is also probably `/etc/redpanda/coprocessor`

Reply:
thats make sense

Reply:
after check this folder, I notice is related to config stuff, and this path is related to the process.

Reply:
Sounds good.